dm/master/server.go

```diff
@@ -347,7 +347,8 @@ func (s *Server) RegisterWorker(ctx context.Context, req *pb.RegisterWorkerReque
 	}
 	log.L().Info("register worker successfully", zap.String("name", req.Name), zap.String("address", req.Address))
 	return &pb.RegisterWorkerResponse{
-		Result: true,
+		Result:    true,
+		SecretKey: s.cfg.SecretKey,
```

Comment:
😂 Do you have a clear boundary that which communication or file access should be protected? If a malicious dm-worker connects to the cluster it will read the secret key.

Reply:
user should use inter-master-worker TLS to protect this case